gitlab/v4/objects/projects.py

```diff
@@ -543,6 +544,15 @@ def transfer_project(self, to_namespace: str, **kwargs: Any) -> None:
             path, post_data={"namespace": to_namespace}, **kwargs
         )
 
+    @cli.register_custom_action("Project", ("to_namespace",))
+    def transfer_project(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Why change the signature from `id` to `*args`?

Not a big deal, but curious.

Reply:
Just wanted to keep the signature completely generic and pass everything to the real function in case things change, probably doesn't make sense here though as we know what to expect. I can change it

Reply:
No need to change it. Just curious. I don't think it hurts anything. Heck making the intellisense not work is maybe a good idea 😁